src/data/set/lattice.lean

```diff
@@ -402,6 +402,11 @@ theorem sInter_pair (s t : set α) : ⋂₀ {s, t} = s ∩ t :=
 
 @[simp] theorem sInter_range (f : ι → set β) : ⋂₀ (range f) = ⋂ x, f x := Inf_range
 
+lemma sUnion_eq_univ_iff {c : set (set α)} :
```

Comment:
The name suggests that this should be stated as `⋃₀ c = @set.univ α ↔ (∀ a, ∃ b ∈ c, a ∈ b)`